scripts/check-yaml.lean

```diff
@@ -24,17 +24,17 @@ def readJsonFile (α) [FromJson α] (path : System.FilePath) : IO α := do
   liftExcept <| fromJson? <|← liftExcept <| Json.parse <|← IO.FS.readFile path
 
 def databases : List (String × String) :=
-  ["undergrad", "overview", "100"].map fun dir =>
+  ["undergrad", "overview", "100", "1000"].map fun dir =>
     (dir ++ ".json",
-      s!"Entries in `docs/{dir}.yaml` refer to declarations that don't exist. \
+      s!"Entries in `docs/{dir}.yaml` refer to N declaration(s) that don't exist. \
         Please correct the following:")
 
 def processDb (decls : ConstMap) : String × String → IO Bool
 | (file, msg) => do
   let lines := ← readJsonFile DBFile file
   let missing := lines.filter (fun l => !(decls.contains l.2))
   if 0 < missing.size then
-    IO.println msg
+    IO.println (msg.replace "N" s!"{missing.size}")
```

Comment:
Surely `String.replace` is not the right way to do this.
Instead, `databases` should just be `List String` (the first component), and then here we print the message with `{dir}` and `N` filled in.

Reply:
Fixed and extracted to #19688.